alphadia/workflow/manager.py

```diff
@@ -657,14 +683,33 @@ def load_classifier_store(self, path=None):
                         torch.load(os.path.join(path, file))
                     )
 
-    def get_classifier(self, available_columns):
+    def get_classifier(self, available_columns, version):
+        """Gets the classifier for a given set of feature columns and version. If the classifier is not found in the store, gets the base classifier instead.
+
+        Parameters
+        ----------
+        available_columns: list
+            List of feature columns
+        version: int
+            Version of the classifier to get
+
+        Returns
+        ----------
+        object
+            Classifier object
+
+        """
         classifier_hash = column_hash(available_columns)
         if classifier_hash in self.classifier_store:
-            classifier = self.classifier_store[classifier_hash]
+            classifier = deepcopy(self.classifier_store[classifier_hash][version])
```

Comment:
so `version` is rather `version_index`?

Reply:
I suppose so. It isn't used with any other meaning though. Do you think it is a problem?